docs/final report/chapters/abstract.tex

```diff
@@ -1,5 +1,12 @@
 \chapter*{Abstract}
 \setheader{Abstract}
 
-abstract here
+There is a problem. A lot of patients of the ADMIRE project have to be analyzed but there is no analysis software simple enough to do this easily. So what do you do when you want to do something which should not require a lot of effort but no software exists to easily achieve this? Exactly, you build software for it.
 
+To build this software we worked according to the SCRUM method and weekly presented a high fidelity prototype. This resulted in a software which, according to previous meetings, has grown to be accurately satisfying the needs of the customer. % lol quest giver
+
+But why do we even care about these analyses? We care because these analyses might help us understand the patients better and thus might make it possible to improve the system and help the patients better with better advice, for example.
```

Comment:
help the patients better with **better advice**, for example.

Beter advies is een voorbeeld van de patienten beter helpen